package.json

```diff
@@ -130,6 +130,7 @@
     "mime-types": "^2.1.22",
     "moment": "2.24.0",
     "nock": "^12.0.3",
+    "node-loader": "^2.0.0",
```

Comment:
I'm not gonna hold up the release on this, but I don't think we're using it. And if we don't need a package we should remove it since otherwise we increase the surface of possible vulnerabilities

Reply:
[`e704bca` (#2461)](https://github.com/nasa/cumulus/pull/2461/commits/e704bca13a98d208ddacaf502fe35541f2f4ab7f)